addons/account/models/partner.py

```diff
@@ -974,6 +975,17 @@ def _deduce_country_code(self):
             country_code = self.vat[:2].upper()
         return country_code
 
+    @api.model
+    def _get_company_registry_placeholder(self, country_code):
+        if country_code == 'FR':
+            return "12345678900001"  # Siren
+        return ""
```

Comment:
That was my first idea but since the l10n_XX module do not depend on `account`, we cannot override an `account` function in the l10n_XX I think :thinking: